frontend/openchat-agent/src/services/dexes/sonic/swaps/sonic.swaps.client.ts

```diff
@@ -8,7 +8,7 @@ import { Principal } from "@dfinity/principal";
 
 const SONIC_INDEX_CANISTER_ID = "3xwpq-ziaaa-aaaah-qcn4a-cai";
 const TEN_MINUTES = 10 * 60 * 1000;
-const ENABLED: boolean = false;
+const ENABLED: boolean = true;
```

Comment:
Just remove this flag

Reply:
Yes but maybe if there was an issue it would be convenient to rollback